pnpm-lock.yaml

```diff
@@ -256,19 +256,6 @@ importers:
       micro: 9.3.5-canary.3
       ngrok: 4.3.3
 
-  apps/tests:
-    specifiers:
-      '@usevenice/app-config': workspace:*
-      openapi-typescript: 6.1.0
-      remeda: 1.6.1
-      tsx: 3.12.2
-    dependencies:
-      '@usevenice/app-config': link:../app-config
-      remeda: 1.6.1
-    devDependencies:
-      openapi-typescript: 6.1.0
-      tsx: 3.12.2
-
```

Comment:
@tonyxiao why does this diff keep showing up?

Reply:
Those are private tests that Tony keeps in a submodule since it currently contains some of his personal tx data to test against. We'll generate some scrubbed data at some point